### DISCUSSION TITLE
is there a way to drop events without `abort`

### DISCUSSION THREAD

**Discussion by tshepang** [vectordotdev/vector#11679] [Q&A] [answered]
I want to be able to use `reroute_dropped`, but don't want certain events reaching re-route destination.

**Comment by spencergilbert** [comment, marked as answer]
Perhaps not exactly what you were thinking of, but perhaps use the events from `reroute_dropped` into a `filter` transform to only route the events you want?

**Reply by tshepang** [reply]
That'd work, only it adds more complexity to my already hairy graph.

**Reply by spencergilbert** [reply]
So you basically want three outputs, happy, dropped but rerouted, and super dropped?

**Reply by tshepang** [reply]
yes

**Reply by tshepang** [reply]
So, maybe we can have `skip` or `ignore` statement, in addition to `abort`.

**Reply by spencergilbert** [reply]
I think `filter` is the way to go for now - but definitely open an issue regarding the `skip`/`ignore` statement!
